graphql/resolvers/contactRequests.js

```diff
@@ -0,0 +1,49 @@
+const { UserInputError } = require("apollo-server");
+
+const contactRequest = require("../../models/ContactRequest.js");
+
+require("dotenv").config();
+
+const { validateContactUsForm } = require("../../util/validators");
+
+module.exports = {
+    Mutation: {
+        async submitContactRequest(_, 
+            { contactUsFormInput: 
```

Comment:
I think I might not want to use a formInput, since I think those take up a lot of space in our type definitions and I want to remove them in other parts of the code too. so instead of expecting a contactUsFormInput you can just expect the variables